doc/classes/ProjectSettings.xml

```diff
@@ -827,6 +827,23 @@
 		<member name="display/window/subwindows/embed_subwindows" type="bool" setter="" getter="" default="true">
 			If [code]true[/code] subwindows are embedded in the main window.
 		</member>
+		<member name="display/window/vsync/buffer_count" type="int" setter="" getter="" default="2">
+			Sets the number of buffers before stalling to wait for the GPU.
+			1 may give you the lowest lag/latency but at the high cost of no parallelism between CPU &amp; GPU.
+			Try the [url=https://darksylinc.github.io/vsync_simulator/]V-Sync Simulator[/url], an interactive interface that simulates presentation to better understand how it is affected by different variables under various conditions.
+			[b]Note:[/b] This setting is not supported by all APIs.
+			[b]Note:[/b] This property is only read when the project starts. There is currently no way to change this value at run-time.
+		</member>
+		<member name="display/window/vsync/swapchain_count" type="int" setter="" getter="" default="3">
+			Sets the number of swapchains (back buffer + front buffer).
```

Comment:
I'm not sure (maybe because of ignorance) about this wording. Isn't there a single swapchin consisting of 1 front buffer plus N back buffers?

Reply:
This is always confusing because D3D11 asks for backbuffer count (backbuffer = 2 means triple buffer because you have to add the implicit front buffer) while Vulkan asks for swapchain count (where swapchaincount = 2 means double buffer because you're including the front one).

Reply:
What do you think about naming this `swapchain_image_count`?

Reply:
I may be fundamentally misunderstanding something, but is it ever actually useful for this to be configurable? The description seems to suggest this is completely determined by other settings. For `VSYNC_DISABLED` the only sensible value is 2. For `VSYNC_ENABLED` and `VSYNC_ADAPTIVE` this feels like it ought to be `buffer_count + 1`. For `VSYNC_MAILBOX` the only sensible value is 3.

Reply:
Ah, I was definitely misunderstanding. Your tool is really useful, and the info in the README cleared everything up for me. The documentation you've added in this PR makes perfect sense. Thank you very much, and sorry for derailing!